### ISSUE TITLE
Feature request: pull-deps

### ISSUE THREAD

**Issue by andrewm-aero** [druid-io/druid-operator#245]
My team is currently investigating the use of this tool, as well as developing some custom druid extensions. It would be nice if we could install those extensions directly through the Druid custom resource spec. Druid includes a command line tool for doing exactly this: https://druid.apache.org/docs/latest/operations/pull-deps.html .

**Comment by pjain1** [comment]
Yeah I don't think this is supported. As of now you will have to do this in entry point thus potentially building your own Druid image. Anyways @himanshug @AdheipSingh can add this to roadmap if they want.
